packages/ui/.storybook/main.ts

```diff
@@ -42,6 +42,15 @@ const config: StorybookConfig = {
   docs: {
     autodocs: "tag",
   },
+  swc: () => ({
+    jsc: {
+      transform: {
+        react: {
+          runtime: "automatic",
+        },
+      },
+    },
+  }),
```

Comment:
why do we need this config?

Reply:
SWC is Storybook's builder, and it doesn't support by default the `jsx-runtime` module that allows you to avoid explicitly import React when you're in a React-based project

With this new configuration rule, we can drop the explicit React imports in all our files without Storybook crashing (due to the above explanation)

For more context: https://storybook.js.org/docs/configure/compilers#the-swc-compiler-doesnt-work-with-react